docs/index.md

```diff
@@ -101,6 +110,32 @@ module.exports = function override(config, env) {
 
 </details>
 
+
+```
++-- your-project
+|   +-- config-overrides.js
+|   +-- node_modules
+|   +-- package.json
+|   +-- public
+|   +-- README.md
+|   +-- src
+```
+
+#### 3) 'Flip' the existing calls to `react-scripts` in `npm` scripts for start, build and test
+```diff
+  /* package.json */
+
+  "scripts": {
+-   "start": "react-scripts start",
++   "start": "react-app-rewired start",
+-   "build": "react-scripts build",
++   "build": "react-app-rewired build",
+    "eject": "react-scripts eject"
```

Comment:
we've lost the "test" line, it should still be here and be black and unchanged like the "eject" line.